subprojects/gradle/utils/src/main/kotlin/com/avito/utils/logging/CILoggingDestination.kt

```diff
@@ -25,7 +25,6 @@ object StdoutDestination : CILoggingDestination() {
         println(formattedMessage.message)
         formattedMessage.cause?.let { throwable ->
             println(throwable.message)
-            throwable.printStackTrace()
```

Comment:
Do we lose a stacktrace now?

Reply:
it's available in sentry

Reply:
It's hard to find all stacktraces related to build
Could we generate a link to the Sentry with applied filters to see all reported error by current build?

Reply:
great idea! will do in separate PR